site/app/_components/AnalyticsProvider.tsx

```diff
@@ -0,0 +1,16 @@
+'use client';
+
+import posthog from 'posthog-js';
+import { PostHogProvider } from 'posthog-js/react';
+
+if (typeof window !== 'undefined') {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
+    api_host: process.env.NEXT_PUBLIC_POSTHOG_HOST,
+    person_profiles: 'identified_only',
```

Comment:
I think we probably want this to be "always" here since we'll never be identifying anyone on this site.

Reply:
updated!